src/common/components/form/fields/MultiDropdownField.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames';
+import { FieldProps } from 'formik';
+import { Select, SelectProps } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { invalidFieldClass } from '../constants';
+import { getErrorText } from '../utils';
+
+export type Option = {
+  label: string;
+  value: string;
+};
+
+type Props = SelectProps<Option> &
+  FieldProps & {
+    defaultValue: Option[];
+    setFieldValue?: (
+      field: string,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      value: any,
+      shouldValidate?: boolean | undefined
+    ) => void;
+  };
+
+const DropdownField: React.FC<Props> = ({
```

Comment:
Name could be MultiDropdownField to make debugging easier

Reply:
fixed